cli_tools/gce_onestep_image_import/onestep_importer/aws_importer.go

```diff
@@ -188,16 +191,70 @@ func (importer *awsImporter) run(args *OneStepImportArguments) error {
 
 	// 4. run image import
 	log.Println("Starting to import image ...")
+	err = importer.importImage(importArgs, startTime, gcsFilePath)
+	if err != nil {
+		return err
+	}
+	log.Println("Image import from AWS finished successfully!")
+
+	// 5. clean up
+	log.Println("Cleaning up ...")
```

Comment:
Let's make it specific: cleanup AWS exported file

Reply:
done

Reply:
Log message still not updated?

Reply:
I think Tony meant to make the comment more specific. The log text messages has been reviewed by Shawnique.